plugins/recruitee/src/dataSources.ts

```diff
@@ -0,0 +1,311 @@
+            id: "location_ids",
+            name: "Locations",
+            type: "multiCollectionReference",
+            collectionId: "",
+            dataSourceId: "Locations",
+        },
+    ]
+)
+const departmentsSchema = v.array(v.object({ departments: v.array(DepartmentSchema) }))
+const departmentsDataSource = createDataSource(
+    {
+        name: "Departments",
+        fetch: async (token: string, companyId: string) => {
+            const url = `https://api.recruitee.com/c/${companyId}/departments`
+            const data = v.safeParse(departmentsSchema, await fetchRecruiteeData(url, token))
+            if (!data.success) {
+                console.log("Error parsing Recruitee data:", data.issues)
+                throw new Error("Error parsing Recruitee data")
+            }
+            return data.output.flatMap(page => page.departments)
+        },
+    },
+    [
+        { id: "id", name: "ID", type: "string", canBeUsedAsSlug: true },
+        { id: "name", name: "Name", type: "string", canBeUsedAsSlug: true },
+        { id: "status", name: "Status", type: "string" },
+        { id: "offers_count", name: "Offers Count", type: "number" },
+        { id: "talent_pools_count", name: "Talent Pools Count", type: "number" },
+    ]
+)
+
+const candidatesSchema = v.array(v.object({ candidates: v.array(CandidateSchema) }))
+const candidatesDataSource = createDataSource(
+    {
+        name: "Candidates",
+        fetch: async (token: string, companyId: string) => {
+            const url = `https://api.recruitee.com/c/${companyId}/candidates`
+            const data = v.safeParse(candidatesSchema, await fetchRecruiteeData(url, token))
+            if (!data.success) {
+                console.log("Error parsing Recruitee data:", data.issues)
+                throw new Error("Error parsing Recruitee data")
+            }
+            return data.output.flatMap(page => page.candidates)
+        },
+    },
+    [
+        { id: "id", name: "ID", type: "string", canBeUsedAsSlug: true },
+        { id: "initials", name: "Initials", type: "string" },
+        { id: "name", name: "Name", type: "string", canBeUsedAsSlug: true },
+        { id: "example", name: "Example", type: "boolean" },
+        {
+            id: "emails",
+            name: "Emails",
+            type: "string",
+            getValue: value => {
+                const list = v.parse(v.array(v.string()), value)
+                return list.join(", ")
+            },
+        },
+        { id: "invalid_emails", name: "Invalid Emails", type: "string" },
+        {
+            id: "phones",
+            name: "Phones",
+            type: "string",
+            getValue: value => {
+                const list = v.parse(v.array(v.string()), value)
+                return list.join(", ")
+            },
+        },
+
+        { id: "positive_ratings", name: "Ratings", type: "number" },
+        { id: "photo_thumb_url", name: "Photo Thumb Url", type: "string" },
```

Comment:
```suggestion
        { id: "photo_thumb_url", name: "Thumbnail Photo", type: "image" },
```

Import thumbnail photos as images. The values are already in the correct format to be synced as images.